source/packaging-guide/discoverability.rst

```diff
@@ -4,34 +4,33 @@ Making your AppImages discoverable
 Now that you have produced your AppImage, you want users to be able to easily find it. To get users, you need to make your AppImage known to the world.
 
 
-.. _application-website:
+.. _ref-application-website:
 
 Application website
 -------------------
 
 The most obvious place to advertise your AppImage is to prominently featured on your application’s download page.
 
 
-.. _social-media:
+.. _ref-social-media:
```

Comment:
Perhaps add the word `discoverability` to be able to distinguish this from future section headings.

I start to believe all these "preparing references" should be removed, and added on demand again.